lib/checkPullRequestLabels.js

```diff
@@ -285,6 +285,15 @@ module.exports.checkChangelogLabel = async function(context) {
       'guidance. Thanks!',
```

Comment:
Update this too - assigning @x to add the label.

Reply:
Done.